src/redux/reducers/index.js

```diff
@@ -1,6 +1,10 @@
 import { combineReducers } from 'redux';
 import blogReducer from './blogReducers';
+import authReducer from './authReducers';
+import { firebaseReducer } from 'react-redux-firebase';
 
 export default combineReducers({
-  blog: blogReducer
+  firebase: firebaseReducer,
```

Comment:
What does this firebase reducer do?

Reply:
firebaseReducer also comes from 'react-redux-firebase' package and it allows me to access data about a specific user. for example: the id of the user.